### PR TITLE
Update 1.1Setup for CDH and Cluster & Client Mode on CDH

### PR DESCRIPTION
**Pull request by ManfeiBai**
Part 4 and 5 should be modified and tested later

### CODE REVIEW

File: docs/readthedocs/source/doc/UserGuide/hadoop.md [hunk 1]
```diff
@@ -42,7 +42,11 @@ You can run Analytics Zoo programs on standard Hadoop/YARN clusters without any
 
 ### **1.1 Setup for CDH**
 
-CDH Version: CDH 5.X, CDH 6.X is not supported
+CDH Version: CDH 5.15.2 is supportted
+CDH Version: other CDH 5.X, CDH 6.X is not supported
+
+- Download and install [CDH](https://docs.cloudera.com/documentation/enterprise/latest/topics/installation.html) and [MariaBD](https://docs.cloudera.com/documentation/enterprise/5-15-x/topics/install_cm_mariadb.html#install_cm_mariadb_config) on your cluster.
```
Comment (qiuxin2012) [inline docs/readthedocs/source/doc/UserGuide/hadoop.md:48]: Delete this. We asume the CDH is installed. CDH installing should be done by Cloudera's service or user's porfessional engineer.

Comment (ManfeiBai) [reply docs/readthedocs/source/doc/UserGuide/hadoop.md]: Done, thanks~

File: docs/readthedocs/source/doc/UserGuide/hadoop.md [hunk 2]
```diff
@@ -125,3 +125,97 @@ Follow the steps below if you need to run Analytics Zoo in [YARN cluster mode](h
   ```
 
   You can adjust the configurations according to your cluster settings.
+  
+  
+  ---
+### **4. YARN Client Mode on CDH**
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/UserGuide/hadoop.md:131]: why do we need a new section? how's it different from section 2?

File: docs/readthedocs/source/doc/UserGuide/hadoop.md [hunk 3]
```diff
@@ -125,3 +125,97 @@ Follow the steps below if you need to run Analytics Zoo in [YARN cluster mode](h
...
+  ```
+
+  or as a normal Python script (e.g. script.py):
+
+  ```bash
+  python script.py
+  ```
+
+---
+### **5. YARN Cluster Mode on CDH**
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/UserGuide/hadoop.md:166]: why do we need a new section? how's it different from section 3?

File: docs/readthedocs/source/doc/UserGuide/hadoop.md (hunk 2 above)
Comment (ManfeiBai) [reply docs/readthedocs/source/doc/UserGuide/hadoop.md:131]: This new section is a draft from section 2. I would delete section 4 and modify section 2 after testing submit scripts on CDH. Thanks~

File: docs/readthedocs/source/doc/UserGuide/hadoop.md (hunk 3 above)
Comment (ManfeiBai) [reply docs/readthedocs/source/doc/UserGuide/hadoop.md:166]: > why do we need a new section? how's it different from section 3?

This new section is a draft from section 3. I would delete section 5 and modify section 3 after testing submit scripts on CDH. Thanks~

Comment (ManfeiBai) [reply docs/readthedocs/source/doc/UserGuide/hadoop.md:117]: Deleted section 5 and section 3 has been modified~

File: docs/readthedocs/source/doc/UserGuide/hadoop.md [hunk 4]
```diff
@@ -42,45 +42,79 @@ You can run Analytics Zoo programs on standard Hadoop/YARN clusters without any
...
-### **2. YARN Client Mode**
+### **2. YARN Client Mode on CDH**
 
-- Install Analytics Zoo in the created conda environment via pip:
+Follow the steps below if you need to run Analytics Zoo in [YARN client mode](https://spark.apache.org/docs/latest/running-on-yarn.html#launching-spark-on-yarn).
+
+- Download and extract [Spark](https://spark.apache.org/downloads.html). You are recommended to use [Spark 2.4.3](https://archive.apache.org/dist/spark/spark-2.4.3/spark-2.4.3-bin-hadoop2.7.tgz). Set the environment variable `SPARK_HOME`:
 
   ```bash
-  pip install analytics-zoo
```
Comment (qiuxin2012) [inline docs/readthedocs/source/doc/UserGuide/hadoop.md:53]: Don't change this, pip and anaconda is recommended for yarn-client, and you had better find another orca example.

Comment (ManfeiBai) [reply docs/readthedocs/source/doc/UserGuide/hadoop.md:53]: > Don't change this, pip and anaconda is recommended for yarn-client, and you had better find another orca example.

Thanks, would modify soon, and adding orca example now~

**Pull request closed without merging**